packages/datadog-plugin-aws-sdk/test/serverless-peer-service.spec.js

```diff
@@ -0,0 +1,363 @@
+              'expected the aws span to have the same peer.service tag as the underlying http span'
+            ).to.equal(httpSpan.meta['peer.service'])
+          })
+
+          await Promise.all([
+            tracesPromise,
+            send({
+              TableName: tableName,
+              Item: {
+                id: { S: '123' }
+              }
+            })
+          ])
+        })
+      })
+
+      describe('Kinesis-Serverless', () => {
+        let kinesis
+        const streamName = 'PeerStream'
+        const kinesisClientName = moduleName === '@aws-sdk/smithy-client' ? '@aws-sdk/client-kinesis' : 'aws-sdk'
+
+        function createStream (cb) {
+          AWS = require(`../../../versions/${kinesisClientName}@${version}`).get()
+
+          const params = {
+            endpoint: 'http://127.0.0.1:4566',
+            region: 'us-east-1'
+          }
+
+          if (moduleName === '@aws-sdk/smithy-client') {
+            const { NodeHttpHandler } = require(`../../../versions/@aws-sdk/node-http-handler@${version}`).get()
+
+            params.requestHandler = new NodeHttpHandler()
+          }
+
+          kinesis = new AWS.Kinesis(params)
+
+          kinesis.createStream({
+            StreamName: streamName,
+            ShardCount: 1
+          }, (err, _) => {
+            if (err) return cb(err)
+
+            helpers.waitForActiveStream(kinesis, streamName, cb)
+          })
+        }
+
+        before(done => {
+          createStream(done)
+        })
+
+        after(done => {
+          kinesis.deleteStream({
+            StreamName: streamName
+          }, (err, res) => {
+            if (err) return done(err)
+
+            helpers.waitForDeletedStream(kinesis, streamName, done)
+          })
+        })
+
+        it('propagates peer.service', done => {
+          agent.assertSomeTraces(traces => {
+            const spans = traces[0]
+            const awsSpan = spans.find(s => s.name === 'aws.request')
+            const httpSpan = spans.find(s => s.name === 'http.request')
+            expect(awsSpan, 'expected the aws span to exist').to.exist
+            expect(httpSpan, 'expected the underlying http span to exist').to.exist
+            expect(awsSpan.meta['peer.service'],
+              'expected the aws span to have a peer.service tag'
+            ).to.exist
+            expect(httpSpan.meta['peer.service'],
+              'expected the underlying http span to have a peer.service tag'
+            ).to.exist
+            expect(awsSpan.meta['peer.service'],
+              'expected the aws span to have the same peer.service tag as the underlying http span'
+            ).to.equal(httpSpan.meta['peer.service'])
+          }, { timeoutMs: 10000 }).then(done, done)
+
+          helpers.putTestRecord(kinesis, streamName, helpers.dataBuffer, e => e && done(e))
+        })
+      })
+
+      describe('SNS-Serverless', () => {
+        let sns
+        let sqs
+        let TopicArn
+        let QueueUrl
```

Comment:
Tiny nit (just less verbose):

```suggestion
        let sns, sqs, TopicArn, QueueUrl
```